README.md

```diff
@@ -1,2 +1,91 @@
 # test-automation
-Tools for making our tests easier to run
+
```

Comment:
I'm summarizing the items that we should be fixing before merging in this:

* [ ] Make sure that enterprise installation works as expected
    * [ ] The branch is manually coded
    * [ ] Output of the following command is confusing, might be a bug there since we're cloning from `citus` repo instaed of enterprise:
```
fab use.citus:v6.0.1 setup.enterprise
[localhost] Executing task 'use.citus'
[localhost] local: rm -rf /home/ec2-user/citus || true
[localhost] local: git clone -q https://github.com/citusdata/citus.git /home/ec2-user/citus
```

* [ ] Make sure that shard rebalancer can be installed. Currently, there is no tasks associated with it.

* [ ] Make sure to document @marcocitus 's comment about key-pair issue on README regarding closed source projects. That worked for me.

* [ ] Make sure that `cstore_fdw` installation updates `postgresql.conf`. It'd be awesome if you create a generic `update_postgres_config` task that we could use for other purposes as well. I had some suggestions on how to do that in the PR.

* [ ] Make sure that postgresql source is complied with `-j ` option. That could decrease the compilation time dramatically.

* [ ] Enable running regression tests for `citus` and `postgres` with a task. And, document them.

* [ ] Try to check in #23. That could save a lot of time for us.


After tackling the above items, could you ping me so that I can do a final testing?

Thanks!